pkg/generate/app/imagestreamlookup.go

```diff
@@ -56,10 +58,20 @@ func (r ImageStreamSearcher) Search(terms ...string) (ComponentMatches, error) {
 				} else {
 					imageref, _ := imageapi.ParseDockerImageReference(term)
 					imageref.Name = stream.Name
+
+					localSearchTag := searchTag
```

Comment:
why is this variable needed?  the original searchTag value is never referenced again after you assign localSearchTag below, so you could have just continued using searchTag.